.circleci/config.yml

```diff
@@ -18,13 +18,16 @@ jobs:
       - run: go get -u golang.org/x/lint/golint
       - run: golint logdna/
   imagest:
```

Comment:
nit: I imagine this should read `imagetest` or `buildtest` here and at the bottom

Reply:
TBH, I had no idea and finding out seemed slow.  I was hesitant to mess with it, but I noticed the same thing.  Gonna leave it.

Reply:
It's `image st` too...not sure what `st` would be

Reply:
I thought it was a typo for `image test`, but really not a blocker at this point with circle being a temp thing

Reply:
I pondered this, and am changing it to `imagetest`.  This word must only be in this local config file.